apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V30__Create_preadjudicated_claims_tables.sql

```diff
@@ -0,0 +1,57 @@
+/*
+ * Creates the RDA FISS pre-adjudicated claims tables required by the application.
+ * https://confluenceent.cms.gov/pages/viewpage.action?spaceKey=MPSM&title=MVP+0.1+Data+Dictionary+for+Replicated+Data+Access+API
+ */
+
+CREATE SCHEMA IF NOT EXISTS "pre_adj";
+
+/*
+ * Where possible column names map directly to fields in the RDA API result message.
+ */
+create table "pre_adj"."FissClaims"
+(
+    "dcn"               varchar(23) not null,
+    "hicNo"             varchar(12) not null,
```

Comment:
I'd suggest sticking with the source names, as @ericbuckley describes. It will make tracing ETL issues down simpler, and also removes the temptation to "get creative" and start coming up with field names ourselves (which only makes more of a mess).